packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -91,7 +91,7 @@ class AndroidWorkflow extends DoctorValidator implements Workflow {
           'Install Android Studio from: https://developer.android.com/studio/index.html\n'
           'On first launch it will assist you in installing the Android SDK components.\n'
           '(or visit https://flutter.io/setup/#android-setup for detailed instructions).\n'
-          'If Android SDK has been installed to a custom location, set \$$kAndroidHome to that location.'
+          'If Android SDK has been installed to a custom location, set \$$kAndroidHome to that location and add it to your PATH.'
```

Comment:
I guess I could get rid of that part if you'd like, I had seen in another issue that someone wanted more clear instructions that ANDROID_HOME needs to be added to your PATH, so I figure I'd add that in this PR as well. Should I just get rid of this change in my PR? Also, nice to meet you.

Reply:
I'd say something like, "and you may want to add it to your PATH environment variable."

Also, please wrap the last line similarly to the other lines so that it looks nice when it prints out.

Reply:
@gspencergoog thanks so much for your input, I've updated the pull request based on your suggestion. Also very nice to meet you!